### PR TITLE
SAK-25099 - Anonymous topics in forums - add defaults to mysql conversion scripts

### PR DESCRIPTION
**Pull request by bbailla2**

### CODE REVIEW

**Comment by ottenhoff** [comment]
okay looks good. thanks.

File: msgcntr/messageforums-hbm/src/sql/mysql/SAK-25099.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
-alter table MFR_TOPIC_T add column (POST_ANONYMOUS bit);
-alter table MFR_TOPIC_T modify column POST_ANONYMOUS bit not null;
-alter table MFR_TOPIC_T add column (REVEAL_IDS_TO_ROLES bit);
-alter table MFR_TOPIC_T modify column REVEAL_IDS_TO_ROLES bit not null;
-alter table MFR_PERMISSION_LEVEL_T add column (IDENTIFY_ANON_AUTHORS bit);
-alter table MFR_PERMISSION_LEVEL_T modify column IDENTIFY_ANON_AUTHORS bit not null;
-update MFR_PERMISSION_LEVEL_T set IDENTIFY_ANON_AUTHORS = 1 where name = 'Owner';
+ALTER TABLE MFR_TOPIC_T ADD COLUMN (POST_ANONYMOUS BIT);
+ALTER TABLE MFR_TOPIC_T MODIFY COLUMN POST_ANONYMOUS BIT NOT NULL DEFAULT 0;
```
Comment (bbailla2) [inline msgcntr/messageforums-hbm/src/sql/mysql/SAK-25099.sql]: Defaults should be set on the 'add' statements, not 'modify'. I will amend.

**Comment by bbailla2** [comment]
Oops, git issues!

**Comment by bbailla2** [comment]
Git issues resolved; defaults are now set on the 'add column' clauses.

**Pull request merged**